python/gui/symbology-ng/qgs25drendererwidget.sip

```diff
@@ -25,14 +25,14 @@ class Qgs25DRendererWidget : QgsRendererWidget
      * @param style
      * @param renderer the mask renderer (will take ownership)
      */
-    static QgsRendererWidget* create( QgsVectorLayer* layer, QgsStyle* style, QgsFeatureRenderer* renderer );
+    static QgsRendererWidget* create( QgsVectorLayer* layer, QgsStyle* style, QgsFeatureRenderer* renderer /Transfer/ ) /Factory/;
```

Comment:
It seems the doxygen comment is wrong, original renderer is not transferred. (Does not happen with other renderer widgets)